UefiPayloadPkg/UefiPayloadPkg.fdf

```diff
@@ -185,6 +186,13 @@ INF MdeModulePkg/Bus/Scsi/ScsiDiskDxe/ScsiDiskDxe.inf
 
 INF FatPkg/EnhancedFatDxe/Fat.inf
 
+#
+# ACPI Support
+#
+
+INF MdeModulePkg/Universal/Acpi/AcpiTableDxe/AcpiTableDxe.inf

```

Comment:
Do we really use this? We use hacks to add BGRT to ACPI RSDP, not the AcpiTableDxe and BootGraphicsResourceTableDxe modules.

Reply:
We don't use it anymore. I thought I needed it at one point to build everything but as it turns out it's not needed now.

Reply:
Deleted